elastic/kibana
# [UII] Add back `fleet_server.inactive` to `fleet_server.unenrolled` to fleet usage telemetry

## Summary

Resolves https://github.com/elastic/kibana/issues/148720

https://github.com/elastic/telemetry/pull/3490 needs to be merged and post-merge action items done before these fields show in telemetry clusters

### Testing

Fleet usage logger:
1. Change the [interval](https://github.com/jen-huang/kibana/blob/ca5af82dd921fd1d12849e78e0f7f0234189a9bf/x-pack/plugins/fleet/server/services/fleet_usage_logger.ts#L59) of the `FleetUsageLogger` to e.g. `1m`.
2. Wait for the log line and ensure that `fleet_server.inactive` and `fleet_server.unenrolled` are included:
```
[2024-05-22T12:03:07.606-07:00][DEBUG][plugins.fleet] Fleet Usage: {"agents_enabled":true,"agents":{"total_enrolled":0,"healthy":0,"unhealthy":0,"offline":0,"inactive":0,"unenrolled":0,"total_all_statuses":0,"updating":0},"fleet_server":{"total_all_statuses":0,"total_enrolled":0,"healthy":0,"unhealthy":0,"offline":0,"updating":0,"inactive":0,"unenrolled":0,"num_host_urls":0},"license_issued_to":"elasticsearch"}
```

Fleet usage sender:
1. Change the [interval](https://github.com/elastic/kibana/blob/ca5af82dd921fd1d12849e78e0f7f0234189a9bf/x-pack/plugins/fleet/server/services/telemetry/fleet_usage_sender.ts#L30) of the `FleetUsageSender` to e.g. `1m`.
2. Wait for the log line and ensure that `fleet_server.inactive` and `fleet_server.unenrolled` are included:
```
[2024-05-22T12:05:17.577-07:00][DEBUG][plugins.fleet] Fleet usage telemetry: {"agents_enabled":true,"agents":{"total_enrolled":0,"healthy":0,"unhealthy":0,"offline":0,"inactive":0,"unenrolled":0,"total_all_statuses":0,"updating":0},"fleet_server":{"total_all_statuses":0,"total_enrolled":0,"healthy":0,"unhealthy":0,"offline":0,"updating":0,"inactive":0,"unenrolled":0,"num_host_urls":0},"packages":[{"name":"nginx","version":"1.20.0","enabled":false},{"name":"system","version":"1.58.0","enabled":false},{"name":"synthetics","version":"1.2.1","enabled":false}],"agent_checkin_status":{"error":0,"degraded":0},"agents_per_policy":[],"agents_per_os":[],"fleet_server_config":{"policies":[]},"agent_policies":{"count":0,"output_types":[]},"agent_logs_panics_last_hour":[],"agent_logs_top_errors":[],"fleet_server_logs_top_errors":[],"license_issued_to":"elasticsearch"}
```

### Checklist

Delete any items that are not applicable to this PR.
